Code/Tools/ProjectManager/Source/ProjectManagerWindow.cpp

```diff
@@ -25,7 +26,9 @@ namespace O3DE::ProjectManager
             setWindowTitle(QString("O3DE %1").arg(tr("Project Manager")));
         }
 
-        ScreensCtrl* screensCtrl = new ScreensCtrl();
+        m_downloadController = new DownloadController();
```

Comment:
Should we parent this so it gets automatically cleaned up?
```suggestion
        m_downloadController = new DownloadController(this);
```

Reply:
I noticed you made this a `QPointer` but doesn't it still need to be parented for it to be destroyed automatically when ProjectManagerWindow is destroyed?

Reply:
Yes it does! Good catch, forgot to parent it here when during moving this creation about.